app/components/sections/problem-solution-section.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import {Grid} from '../grid'
+import {H2, H3} from '../title'
+import {ArrowButton} from '../arrow-button'
+import {ArrowIcon} from '../icons/arrow-icon'
+import {Paragraph} from '../paragraph'
+
+function ProblemSolutionSection() {
+  const arrow = (
+    <span className="h-[50px] hidden items-center mt-4 lg:flex">
+      <ArrowIcon size={76} direction="right" />
+    </span>
+  )
+
+  return (
+    <div className="px-8 w-full">
+      <div className="pb-16 pt-24 w-full bg-gray-100 dark:bg-gray-800 rounded-lg lg:pb-40 lg:pt-36">
+        <div className="-mx-8">
+          <Grid>
+            <div className="col-span-full lg:col-span-5">
+              <H2>
+                The JavaScript landscape is incredibly confusing to keep up with
+                right?
+              </H2>
+            </div>
+            <div className="col-span-full lg:col-span-5 lg:col-start-7">
+              <H2 variant="secondary">
+                You’re in the right place, my website is your one stop shop for
+                everything JavaScript related.
+              </H2>
+            </div>
+
+            <hr className="col-span-full mb-16 mt-20 border-gray-200 dark:border-gray-600" />
+
+            <div className="col-span-full order-2 mt-16 lg:col-span-5 lg:col-start-7 lg:mt-0">
+              <img className="-ml-10 w-48" src="/placeholders/skis.png" />
+            </div>
+
+            <div className="col-span-full col-start-1 order-1 lg:col-span-5 lg:order-3">
+              <ul className="flex flex-row text-white text-xl space-x-8 lg:flex-col lg:text-7xl lg:space-x-0 lg:space-y-7">
+                <li>
+                  <a
```

Comment:
I'm unsure what this menu is about, or what it does. It has an arrow, and it has 1 bright and 2 dimmed links. What does it do?

Reply:
I *think* that these should be `<button>`s and clicking them will update the info on the right. Could/should probably use [`@reach/tabs`](https://reach.tech/tabs) for this.

Reply:
That idea had crossed my mind. Let's make it like that.

Reply:
Would be cool if when you changed the selection there was a tiny animation of the arrow to move slightly from the left to the right

Reply:
Done. Might need some fine-tuning in speed and distance, but that's trivial.

https://github.com/kentcdodds/remix-kentcdodds/blob/f31b3fefa09fc56cac7768b2574999e2b5b60f20/app/components/sections/problem-solution-section.tsx#L33-L35